### PR TITLE
DEVOPS-817 Adding composite action for tf scan

### PR DESCRIPTION
**Pull request by jorge-cr-13**
## What 🕵️ 🔍

- Adding a composite action to reduce the error prompt file for TF scan. 

## References 📝 🔗

- [Jira task](https://xainag.atlassian.net/jira/software/projects/DEVOPS/boards/65?selectedIssue=DEVOPS-816)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Update(s) to the configuration of existing service(s)
- [ ] This change requires a documentation update
- [ ] Removed some deprecated application or functionality

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

## Checklist:
 
- [x] I have made corresponding changes to the documentation.
- [x] I have performed a self-review of my own code.
- [x] I did not commit anything sensitive like keys or secrets.
- [x] I added the corresponding labels.
- [x] I set the reviewers and the assignees.
- [x] All open comments and discussions have been resolved.
- [x] My branch is following the name pattern: `JIRA-ID_Short_descripton`, example: `DEVOPS-000_some_meaningful_description`

### CODE REVIEW

**Review by APB-s — COMMENTED** [review]
I simplify the code by replacing specific paths by "pathof/change/files/**".
Tested and functionnal

File: .github/workflows/tfsec_sarif_report.yml [hunk 1]
```diff
@@ -1,115 +1,69 @@
...
-        uses: aquasecurity/tfsec-sarif-action@v0.1.0
-        with:
-          working_directory: aws/dns/domains/
-          sarif_file: tfsec.sarif
-
-      - name: Upload SARIF file
-        uses: github/codeql-action/upload-sarif@v2
-        with:
-          sarif_file: tfsec.sarif 
+          sarif_file: tfsec.sarif
```
Comment (abargiela) [inline .github/workflows/tfsec_sarif_report.yml:69]: github is complaining about a missing new line, could you please include it?

Comment (APB-s) [reply .github/workflows/tfsec_sarif_report.yml]: Sure, I do this

**Review by APB-s — COMMENTED** [review]
Line added, comments added

**Pull request merged**